server/graphql/v1/mutations/orders.js

```diff
@@ -68,6 +80,30 @@ export async function createOrder(order, loaders, remoteUser) {
       );
     }
 
+    // Check if FromCollective reached max limit of orders per hour
+    if (ordersFromCollectiveCache.get(order.fromCollective.id)) {
+      if (ordersFromCollectiveCache.get(order.fromCollective.id) > 10) {
+        debug('Orders Cache: You have reached the hourly limit of donations', order);
+        throw new Error('You have reached the hourly limit of donations');
+      }
+      ordersFromCollectiveCache.set(order.fromCollective.id, ordersFromCollectiveCache.get(order.fromCollective.id) + 1);
```

Comment:
What happens if the value is undefined? I would suggest to explicitly set it to 0.

Reply:
I agree with all your other comments, but in this one I disagree as:
1) We are already creating the order so I'd set to 1 and not 0
2) I check whether the value for that key exists, if it doesnt I'm setting the new key to one(look the if else).